core/arch/arm/kernel/wait_queue.c

```diff
@@ -110,8 +108,7 @@ void wq_wait_final(struct wait_queue *wq, struct wait_queue_elem *wqe,
 		wq_rpc(OPTEE_MSG_RPC_WAIT_QUEUE_SLEEP, wqe->handle,
 		       sync_obj, owner, fname, lineno);
 
-		old_itr_status = thread_mask_exceptions(THREAD_EXCP_ALL);
-		cpu_spin_lock(&wq_spin_lock);
+		old_itr_status = cpu_spin_lock_xsave(&wq_spin_lock);
```

Comment:
is this replace `thread_mask_exceptions()`+`cpu_spin_lock()` with new `cpu_spin_lock_xsave()`, please update also the counter part line 117+118 with `cpu_spin_lock_xrestore()`.